examples/reference/nuxt.config.js

```diff
@@ -144,7 +144,7 @@ export default {
       username: process.env.CONTENT_API_AUTH_USER,
       password: process.env.CONTENT_API_AUTH_PASS
     },
-    site: 8888,
+    site: 4,
```

Comment:
the test branch was old and it was in content-vic, that is why it needed to point to that. 
Revert back before merging

Reply:
@MdNadimHossain prefer that this is done in Lagoon CLI these days FYI

Reply:
hey @dylankelly 
I don't see any vars in the CMDB though - https://github.com/dpc-sdp/sdp-cmdb/blob/master/config/ripple/config.yml
Will raise it with devops if we can manage it from there may be :/